addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -298,14 +298,14 @@ odoo.define('website_slides.fullscreen', function (require) {
         * @param {Object} slides Contains the list of all slides of the course
         * @param {integer} defaultSlideId Contains the ID of the slide requested by the user
         */
-        init: function (el, slides, defaultSlideId){
+        init: function (el, slides, defaultSlideId, channelData){
             var result = this._super.apply(this,arguments);
             this.initialSlideID = defaultSlideId;
             this.slides = this._preprocessSlideData(slides);
-
+            this.channel = channelData;
             var slide;
             if (defaultSlideId) {
-                slide = findSlide(this.slides, {id: defaultSlideId});
+                slide = findSlide(this.slides, {id: defaultSlideId, isQuiz: window.location.href.indexOf('quiz=1') !== -1});
```

Comment:
to handle url param, you should use `$.param` and `$.deparam` from BBQ lib. See http://benalman.com/projects/jquery-bbq-plugin/.